lib/ae_mdw/db/sync/contract.ex

```diff
@@ -116,4 +109,43 @@ defmodule AeMdw.Db.Sync.Contract do
         end
     end
   end
+
+  defp oracle_and_name_mutations(events, {height, _mbi} = block_index, call_txi) do
+    events
+    |> Enum.filter(fn
+      {{:internal_call_tx, "Oracle.register"}, _info} -> true
```

Comment:
aren't there other oracle mutations that might happen, e.g. extend?

Reply:
We haven't been doing anything about internal calls for oracles, I added the Oracle.register last week. I created an issue for handling Oracle.respond as well: https://github.com/aeternity/ae_mdw/issues/468

Reply:
okay, then I am fine for that, thanks

Reply:
Yes, this issue is only for the AENS. @sborrazas is already working on the other Oracle.*. Thanks